test-pod/pod/test_pod.clj

```diff
@@ -127,7 +128,8 @@
                                                 {"name" "other-tag"}
                                                 ;; reads thing with other tag
                                                 {"name" "read-other-tag"
-                                                 "code" "(defn read-other-tag [x] [x x])"}
+                                                 "code" "(defn read-other-tag [x] [x x])"
+                                                 "meta" "{:doc \"unread\"}"}
```

Comment:
We shouldn't have to support this field, in that case people can just write `(defn ^{:doc "foo"} read-other-tag [])` or `(defn read-other-tag "foo [])`.